src/app/onboarding/components/archive-type-select/archive-type-select.component.ts

```diff
@@ -0,0 +1,33 @@
+/* format */
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+import { archiveOptions, archiveDescriptions } from '@root/app/onboarding/shared/onboarding-screen';
+
+@Component({
+  selector: 'pr-archive-type-select',
+  templateUrl: './archive-type-select.component.html',
+  styleUrls: ['./archive-type-select.component.scss'],
+})
+export class ArchiveTypeSelectComponent {
+  @Input() selectedValue: string = '';
```

Comment:
Since this is always going to deal with Archive Type, this should really return the `ArchiveType` type that can be found in the `archive-vo.ts` models file instead of a string to make it more type safe.

Reply:
@meisekimiu this is a concatenation of the archive type and the tag for mail chimp which is being handled in `create-new-archive` component. I needed both values from the dropdown.

Reply:
Right, I forgot this was being generated from a `<select>` tag. Still, I feel like the type we're emitting from this component shouldn't be just a formatted string, so that the rest of the onboarding flow doesn't have to work around the concatenation format from this component when it's creating the archive and sending mailchimp tags.

Reply:
@meisekimiu I have updated the Output, what do you think?

Reply:
That's looks better. Thanks for updating this. I think ideally it'd be nice to constrain the tag type as well, but I can also see how the tag types might change over time, so it's probably fine to leave it as a string for now.